### PR TITLE
Use torch.testing.assert_close in common_utils.py

### PR DESCRIPTION
**Pull request by NicolasHug**
part of https://github.com/pytorch/vision/pull/3865

### CODE REVIEW

File: test/common_utils.py
Comment (NicolasHug) [inline test/common_utils.py:144]: for ref `self.assertEqual` is currently setting `prec` to `self.precision` if it's None (as done above)

and the check was

```
tolerance = prec + prec * abs(a.max())
self.assertLessEqual(max_err, tolerance, message)
```

So the changes LGTM here

**Review by NicolasHug — COMMENTED** [review]
I can't approve as I'm the PR author, but this is @pmeier 's work, LGTM when green

**Review by datumbox — APPROVED** [review]
Stamping

**Pull request merged**
